apps/server/src/modules/learnroom/controller/dashboard.controller.spec.ts

```diff
@@ -111,7 +111,7 @@ describe('dashboard uc', () => {
 				return Promise.resolve(dashboard);
 			});
 			const currentUser = { userId: 'userId' } as ICurrentUser;
-			await controller.findForUser(currentUser, true);
+			await controller.findForUser(currentUser, { showSubstitute: true });
 
 			expect(spy).toHaveBeenCalledWith('userId', true);
 		});
```

Comment:
showDashboard is now optional, so it makes sense to have a test where this is not set at all
Api tests (e2e) are prefered instead of controller tests